packages/manager/apps/sign-up/src/form/component.js

```diff
@@ -14,5 +14,7 @@ export default {
     isSmsConsentAvailable: '<',
     needkyc: '<',
     goToKycDocumentUploadPage: '<',
+    user: '<',
+    subsidiary: '<',
```

Comment:
Why just not add `subsidiary` with the value `this.subsidiary || this.user.subsidiary`? You don't really need whole `user`